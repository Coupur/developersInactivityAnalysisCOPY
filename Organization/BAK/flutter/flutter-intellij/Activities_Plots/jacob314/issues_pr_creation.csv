id,date,creator_login
266233410,2017-10-17 18:36:14,jacob314
266311311,2017-10-17 23:23:07,jacob314
277579351,2017-11-28 23:45:48,jacob314
277847712,2017-11-29 17:48:22,jacob314
278608994,2017-12-01 21:59:01,jacob314
279461637,2017-12-05 17:26:21,jacob314
279994943,2017-12-07 04:10:48,jacob314
285474698,2018-01-02 16:32:04,jacob314
285474850,2018-01-02 16:32:39,jacob314
285474983,2018-01-02 16:33:09,jacob314
285475113,2018-01-02 16:33:38,jacob314
285475522,2018-01-02 16:35:14,jacob314
285476120,2018-01-02 16:37:35,jacob314
285476865,2018-01-02 16:40:22,jacob314
285477369,2018-01-02 16:42:23,jacob314
285478061,2018-01-02 16:44:58,jacob314
285478187,2018-01-02 16:45:28,jacob314
285478342,2018-01-02 16:46:07,jacob314
285478453,2018-01-02 16:46:30,jacob314
289469336,2018-01-18 01:08:11,jacob314
292531055,2018-01-29 19:29:46,jacob314
292943232,2018-01-30 21:27:03,jacob314
293674070,2018-02-01 20:34:02,jacob314
293675037,2018-02-01 20:37:31,jacob314
293677132,2018-02-01 20:44:40,jacob314
293714120,2018-02-01 22:57:12,jacob314
293715377,2018-02-01 23:02:31,jacob314
293742276,2018-02-02 01:24:54,jacob314
293976875,2018-02-02 18:12:42,jacob314
293980065,2018-02-02 18:24:59,jacob314
297637421,2018-02-15 23:44:55,jacob314
299565733,2018-02-23 01:09:34,jacob314
300723314,2018-02-27 17:28:36,jacob314
301128872,2018-02-28 18:15:39,jacob314
301602574,2018-03-01 23:46:16,jacob314
305335751,2018-03-14 21:29:01,jacob314
307463166,2018-03-22 00:20:49,jacob314
307747930,2018-03-22 17:43:12,jacob314
311419092,2018-04-04 22:57:29,jacob314
312779214,2018-04-10 05:36:48,jacob314
314865230,2018-04-17 00:40:34,jacob314
315158620,2018-04-17 17:29:03,jacob314
316420589,2018-04-20 21:43:57,jacob314
316619786,2018-04-22 22:40:10,jacob314
317017048,2018-04-23 23:24:35,jacob314
317041383,2018-04-24 01:46:45,jacob314
317054911,2018-04-24 03:12:29,jacob314
317304673,2018-04-24 16:32:34,jacob314
317433050,2018-04-24 23:46:04,jacob314
318169082,2018-04-26 19:37:21,jacob314
318170857,2018-04-26 19:42:45,jacob314
318450066,2018-04-27 15:20:50,jacob314
320983914,2018-05-07 22:44:32,jacob314
320991746,2018-05-07 23:23:22,jacob314
321004445,2018-05-08 00:38:02,jacob314
327847965,2018-05-30 18:28:44,jacob314
329149832,2018-06-04 17:33:39,jacob314
330824658,2018-06-08 22:53:33,jacob314
331376705,2018-06-11 22:45:43,jacob314
332483893,2018-06-14 16:51:46,jacob314
336005317,2018-06-26 21:55:53,jacob314
342900038,2018-07-19 21:43:47,jacob314
346245884,2018-07-31 15:42:30,jacob314
346260822,2018-07-31 16:19:25,jacob314
348035025,2018-08-06 18:32:15,jacob314
359119361,2018-09-11 16:14:21,jacob314
363208494,2018-09-24 15:51:15,jacob314
363340257,2018-09-24 22:14:22,jacob314
364207911,2018-09-26 20:55:16,jacob314
365523150,2018-10-01 15:43:24,jacob314
366137950,2018-10-03 01:13:59,jacob314
366147225,2018-10-03 02:06:22,jacob314
366407971,2018-10-03 15:51:44,jacob314
366425827,2018-10-03 16:34:20,jacob314
371735310,2018-10-18 22:17:06,jacob314
373172158,2018-10-23 19:52:12,jacob314
375182038,2018-10-29 19:35:58,jacob314
375230155,2018-10-29 21:45:53,jacob314
375582107,2018-10-30 16:32:07,jacob314
376249315,2018-11-01 04:26:01,jacob314
376249498,2018-11-01 04:27:08,jacob314
376251210,2018-11-01 04:38:18,jacob314
377944020,2018-11-06 17:02:54,jacob314
378469789,2018-11-07 21:08:34,jacob314
379867871,2018-11-12 16:57:55,jacob314
382481406,2018-11-20 03:11:12,jacob314
382489721,2018-11-20 03:52:28,jacob314
382866825,2018-11-20 22:04:24,jacob314
382896117,2018-11-20 23:52:49,jacob314
382932701,2018-11-21 02:50:04,jacob314
382970217,2018-11-21 06:10:13,jacob314
384451392,2018-11-26 18:15:37,jacob314
384468393,2018-11-26 19:06:12,jacob314
384614897,2018-11-27 04:17:02,jacob314
384621110,2018-11-27 04:46:00,jacob314
384877224,2018-11-27 16:24:27,jacob314
384944650,2018-11-27 19:12:25,jacob314
384955499,2018-11-27 19:42:35,jacob314
385034345,2018-11-27 23:33:39,jacob314
385044805,2018-11-28 00:16:47,jacob314
385074082,2018-11-28 02:33:39,jacob314
388014834,2018-12-06 01:15:43,jacob314
388053881,2018-12-06 04:39:33,jacob314
389851036,2018-12-11 16:37:17,jacob314
400100654,2019-01-17 03:58:29,jacob314
408987784,2019-02-11 21:37:10,jacob314
424440407,2019-03-23 00:08:32,jacob314
426119248,2019-03-27 18:28:07,jacob314
426184890,2019-03-27 21:09:49,jacob314
426187208,2019-03-27 21:16:01,jacob314
426188486,2019-03-27 21:19:24,jacob314
426197875,2019-03-27 21:45:48,jacob314
426201537,2019-03-27 21:56:24,jacob314
426222579,2019-03-27 23:10:22,jacob314
426236384,2019-03-28 00:09:29,jacob314
426681858,2019-03-28 20:01:12,jacob314
430560161,2019-04-08 17:05:10,jacob314
435892052,2019-04-22 21:04:59,jacob314
436892322,2019-04-24 20:29:38,jacob314
437875129,2019-04-26 23:08:32,jacob314
438953387,2019-04-30 20:03:15,jacob314
439041558,2019-05-01 01:33:04,jacob314
439690808,2019-05-02 16:56:28,jacob314
440245728,2019-05-03 22:29:40,jacob314
441462052,2019-05-07 21:52:03,jacob314
441503850,2019-05-08 00:52:18,jacob314
441505732,2019-05-08 01:02:47,jacob314
446295119,2019-05-20 20:26:37,jacob314
454919424,2019-06-11 21:49:34,jacob314
462888947,2019-07-01 20:08:33,jacob314
