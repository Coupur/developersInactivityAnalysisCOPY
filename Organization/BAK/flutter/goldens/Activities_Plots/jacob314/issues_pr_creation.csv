id,date,creator_login
350864876,2018-08-15 15:55:15,jacob314
354824373,2018-08-28 17:11:39,jacob314
357337238,2018-09-05 17:51:57,jacob314
